lynx/styles/blue.json

```
@@ -0,0 +1,41 @@
+{
+	"version": 2,
+	"title": "Blue",
+	"settings": {
+		"color": {
+			"palette": [
+				{
+					"slug": "foreground",
+					"color": "#3F67C6",
+					"name": "Foreground"
+				},
+				{
+					"slug": "background",
+					"color": "#FCF5ED",
+					"name": "Background"
+				},
+				{
+					"slug": "primary",
+					"color": "#385BAD",
+					"name": "Primary"
+				},
+				{
+					"slug": "secondary",
+					"color": "#F7EBDD",
+					"name": "Secondary"
+				},
+				{
+					"slug": "tertiary",
+					"color": "#ffffff",
+					"name": "Tertiary"
+				}
+			]
+		}
+	},
+	"styles": {
+		"color": {
+			"background": "var(--wp--preset--color--background)",
+			"text": "var(--wp--preset--color--foreground)"
+		}
+	}
```

```suggestion
	}
```

These are the same settings as the base theme, so you shouldn't need to redefine them in the style variation.

---

Nicely spotted!